### DISCUSSION TITLE
Actions workflow sidebar is too narrow

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#127407] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

Consider this workflow run:

https://github.com/PowerDNS/pdns/actions/runs/9384189812
<img width="1625" alt="image" src="https://github.com/community/community/assets/2119212/63b6d890-9883-4944-bd17-ee4b3af43ed6">

This is because the `--Layout-pane-width` is fairly small. I think at its largest it's <400px.

If it can be set to 580px, the rendering is much nicer:
<img width="1061" alt="image" src="https://github.com/community/community/assets/2119212/c6db2816-6e41-4265-8e81-5e80760aefde">

The right pane can be zoomed/panned, but the sidebar can't be scrolled which is a real problem.

Requests:
* Add tooltips to job names (at least when clipping)
* Allow the left pane to be panned if absolutely necessary
* Allow the splitter between left and right panes to be dragged (preferably persisting per browser session, but at least for a page interaction -- which would include clicking a different job in the loaded "page")

**Comment by grandmas-favorite** [comment, marked as answer]
Hi @jsoref!

We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our public roadmap. While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress.

While we do review all incoming feedback unfortunately due to the volume of feedback, we may not be able to respond to each one individually at this time. However, we strongly encourage you to continue providing your valuable input as it greatly helps us prioritize and improve GitHub.

Your contributions are instrumental in making GitHub better for all users!

**Reply by jsoref** [reply]
@grandmas-favorite: that's fine -- and more or less in line with my expectations -- this isn't a firedrill ("drop everything"). Is it reasonable for me to mark your response as the answer to say "we've heard (but there's no committed timeline)"

**Reply by grandmas-favorite** [reply]
Hey @jsoref! I'd say that's reasonable, but totally up to you 😁 - and thanks again for supplying the feedback, it doesn't go unnoticed!
